eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/diagram/binding/AbstractSYBindingComposite.java

```diff
@@ -246,4 +253,27 @@ protected void doExecute() {
         }
     }
 
+    protected boolean isVersion21OrHigher() {
+        String version = null;
+        if (getTargetObject() != null) {
+            URI uri = getTargetObject().eResource().getURI();
+            IFile tempFile = ResourcesPlugin.getWorkspace().getRoot().getFile(new Path(uri.toPlatformString(true)));
```

Comment:
Did you already tried it with a project stored outside of the workspace location?